gapic/utils/case.py

```diff
@@ -45,3 +45,17 @@ def to_snake_case(s: str) -> str:
 
     # Done; return the camel-cased string.
     return s.lower()
+
+def to_camel_case(s: str) -> str:
```

Comment:
Are there any built-in/standard python functions which do the same? Please prefer using standard ones to custom, if there are any.

Reply:
I'm fairly certain there is no standard library function for to camel-case.